chalicelib_cgap/check_schedules.py

```diff
@@ -102,6 +107,13 @@ def check_runner(event, context):
     the checks. Self propogates. event is a dict of information passed into
     the lambda at invocation time.
     """
+    print('xyzzy/check_runner/enter')
+    print(type(event))
+    print(event)
+    print(type(context))
+    print(context)
```

Comment:
Remove these before merge, maybe keeping one debug statement as that may generally be useful